src/components/Client.tsx

```diff
@@ -72,7 +72,11 @@ export const Client: FC<Props> = ({ clientID, onRemoveClientID }) => {
             {getFirstLetters({ firstName: client.general.firstName, lastName: client.general.lastName })}
           </Avatar>
           <div>
-            <Description bolt={true} title="Name" value={`${client.general.firstName} ${client.general.lastName}`} />
+            <Description
+              boltTitle={true}
```

Comment:
`boltTitle={true}` change to `boltTitle`

Reply:
Done